cppcache/src/Log.cpp

```diff
@@ -726,13 +726,7 @@ void Log::exitFn(LogLevel level, const char* functionName) {
 }
 
 bool Log::enabled(LogLevel level) {
-#ifdef DEBUG
```

Comment:
On closer inspection of the original code you will notice that nothing is defining `DEBUG` in the preprocessor so `s_doingDebug` was always false. This latest change introduces a bug to the logic overrode the results of `GEODE_HIGHEST_LOG_LEVEL >= level`, which appears to be redundant to the `DEBUG` macro. If you rip all that out then you end up with what's in this PR.